### PR TITLE
Xsiam layout rule

### PR DESCRIPTION
**Pull request by eyalpalo**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Tests/scripts/collect_tests/constants.py [hunk 1]
```diff
@@ -106,6 +107,7 @@
     FileType.XDRC_TEMPLATE_YML,
     FileType.XSIAM_DASHBOARD_IMAGE,
     FileType.XSIAM_REPORT_IMAGE,
+
```
Comment (dorschw) [inline Tests/scripts/collect_tests/constants.py:110]:
```suggestion
```

File: Tests/Marketplace/marketplace_services.py [hunk 2]
```diff
@@ -2327,6 +2327,19 @@ def collect_content_items(self):
                             'marketplaces': content_item.get('marketplaces', ["marketplacev2"]),
                         })
 
+                    elif current_directory == PackFolders.LAYOUT_RULES.value and pack_file_name.startswith(
+                            "external-"):
```
Comment (dantavori) [inline Tests/Marketplace/marketplace_services.py:2331]: do we need the external prefix for this type?

File: Tests/Marketplace/marketplace_constants.py [hunk 3]
```diff
@@ -221,6 +221,7 @@ class PackFolders(enum.Enum):
     TRIGGERS = 'Triggers'
     WIZARDS = 'Wizards'
     XDRC_TEMPLATES = 'XDRCTemplates'
+    LAYOUT_RULES = 'LayoutRules'
```
[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Tests/Marketplace/marketplace_constants.py:224]: Regarding the `SIEM_RULES_OBJECTS` var - need to check if we want to mark a pack as a siem pack (and therefore tag it as data source) if it has only a layout rule

[the remark this answers is not in the document]
Comment (dantavori) [reply Tests/Marketplace/marketplace_constants.py:224]: any news about this one @eyalpalo ?

**Review by dantavori — APPROVED** [review]

File: Tests/Marketplace/marketplace_constants.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (eyalpalo) [reply Tests/Marketplace/marketplace_constants.py:224]: yeah, after talking with @bziser and @yucohen decided to remove it from SIEM_RULES_OBJECTS, as, similar to triggers, it is not a SIEM object.

**Review by dantavori — APPROVED** [review]

**Pull request merged**